src/components/ColorSelector.js

```diff
@@ -0,0 +1,62 @@
+import React from "react";
+
+const colors = {
+  items: 28,
+  cssColors: [
+    "Black",
+    "DarkGrey",
+    "DarkRed",
+    "Olive",
+    "Green",
+    "SteelBlue",
+    "DarkBlue",
+    "Purple",
+    "DarkKhaki",
+    "LightGreen",
+    "RoyalBlue",
+    "Indigo",
+    "Navy",
+    "Saddlebrown",
+    "White",
+    "LightGrey",
+    "Red",
+    "Yellow",
+    "Lime",
+    "Skyblue",
+    "Blue",
+    "Violet",
+    "Khaki",
+    "LawnGreen",
+    "Skyblue",
+    "DarkViolet",
+    "Fuchsia",
+    "Chocolate"
+  ]
+};
+
+const ColorSelector = () => {
+  let items = [];
+
+  for (let i = 0; i < colors.items; i++) {
```

Comment:
You are comparing a number with an array?

Reply:
Nop, colors.items is a number (colors.items=28) and items is an array that it is filled inside the for.

Reply:
Why not using colors.cssColors.length?